lib/components/tab.vue

```diff
@@ -36,7 +36,7 @@
         data() {
             return {
                 localActive: this.active && !this.disabled,
-                show: false
+                show: Boolean(this.active && !this.disabled)
```

Comment:
Hmmm... mounted may work, although it should maybe look at localActive on mounted (depending on if b-tabs is set to `lazy` or not.